website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+- **Behavior change:** New behavior change flags for [`state:modified`](/reference/global-configs/behavior-changes#source-definitions-and-variables-for-state) to reduce false positives/negatives and enhanced behavior for [`on-run-start` hooks](/reference/global-configs/behavior-changes#on-run-start-hook) to skip nodes on failure. 
```

Comment:
I think we will want to split this up into a release note per flag and describe what the flag does

Reply:
Good call! I split them up @graciegoheen